wom/src/main/scala/wom/callable/Callable.scala

```diff
@@ -78,14 +78,18 @@ object Callable {
     }
   }
 
+  sealed trait HasDefault extends InputDefinition {
```

Comment:
We'd have to rename the class that's currently named `InputDefinitionWithDefault`, giving us something like
```
sealed trait InputDefinitionWithDefault extends InputDefinition
final case class OverridableInputDefinitionWithDefault extends InputDefinitionWithDefault
final case class FixedInputDefinitionWithDefault extends InputDefinitionWithDefault
```
Is this what you have in mind?

Reply:
(FWIW I like it but would welcome suggestions for more concise names)

Reply:
Hmm, good point... I think `FixedInputDefinition` would be OK since it sort of implies the `WithDefault`, but otherwise I think the suggested names are reasonable (perhaps improvable with effort, but diminishing returns etc...)